src/plugins/cosmos/CosmosAssetAccountDetails.tsx

```diff
@@ -34,7 +57,9 @@ export const CosmosAssetAccountDetails = ({ assetId: caip19, accountId }: AssetD
           <AssetChart accountId={accountId} assetId={caip19} isLoaded={true} />
           {accountId && <AccountAssets assetId={caip19} accountId={accountId} />}
           <AssetAccounts assetId={caip19} accountId={accountId} />
-          {cosmosInvestorFlag && <StakingOpportunities />}
+          {cosmosInvestorFlag && (
+            <StakingOpportunities stakingOpportunities={stakingOpportunities} />
```

Comment:
Since the component is called `<StakingOpportunities />`, you can just name the prop `opportunities`